tox.ini

```diff
@@ -298,32 +298,64 @@ commands =
     bash -c 'HAGRID_ART=false hagrid land test_domain_2'
     bash -c 'HAGRID_ART=false hagrid land test_domain_3'
 
-[testenv:stack.test.integration.k8s]
-description = Integration Tests for Core Stack
+[testenv:stack.test.integration.k8s.network]
+description = Integration Tests for Network Node on Kubernetes
 deps =
     {[testenv:syft]deps}
 changedir = {toxinidir}
 passenv=HOME
 allowlist_externals =
     devspace
+    docker
     kubectl
     minikube
     grep
     sleep
     bash
     kubectx
+setenv =
+  TEST_NETWORK_DEPLOYED_IN = minikube
+  TEST_DOMAIN_DEPLOYED_IN = minikube
+  HAGRID_ART = false
 commands =
     minikube version
-    bash -c 'minikube status && echo "minikube running" || echo "minikube starting" && minikube start --disk-size=40g'
-    minikube addons enable ingress
     devspace --version
-    bash -c 'kubectl create namespace openmined || true'
-    devspace use namespace openmined
-    bash -c 'cd packages/grid && devspace deploy -b -p domain'
-    bash -c 'while ! (kubectl get ingress &) | grep -Eq "stack-ingress.*[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}"; do echo "waiting for ingress" && sleep 1; done'
 
-    pytest tests/integration -m k8s -p no:randomly --co
-    bash -c 'TEST_DOMAIN_IP=`minikube ip` TEST_DOMAIN_PORT=80 pytest tests/integration -m k8s -vvvv -p no:randomly -p no:benchmark -o log_cli=True --capture=no'
+    bash -c 'minikube start --profile=syft-network --disk-size=40g'
+    bash -c 'kubectl --context syft-network create namespace openmined || true'
+    bash -c 'cd packages/grid && devspace --kube-context syft-network --namespace openmined deploy -b -p network -p node-port -p no-frontend'
+
+    bash -c 'minikube start --profile=syft-domain --disk-size=40g'
+    bash -c 'kubectl --context syft-domain create namespace openmined || true'
+    bash -c 'cd packages/grid && devspace --kube-context syft-domain --namespace openmined deploy -b -p domain -p node-port'
+    docker network connect syft-network syft-domain
```

Comment:
This part may not be so simple. I will experiment with this locally with some smaller minikube clusters so that I can actually run 2 of them.